src/Common/src/Interop/Windows/user32/Interop.MessageBox.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class User32
```

Comment:
Correct.  These are not exposed via mincore.  Later on we use lightup to fail in the case where the API is not present.